decoder/rubix.go

```diff
@@ -9,10 +9,13 @@ import (
 	"unsafe"
 
 	"github.com/NubeIO/nubeio-rubix-lib-models-go/model"
+	log "github.com/sirupsen/logrus"
 )
 
 const HEADER_BIT_COUNT = 6
 
+var position uint8 = 0
```

Comment:
move to local variable of decode func instead of global. This is also never reset to 0 for subsequent payloads coming in